node/nodejs_intro/README.md

```diff
@@ -69,55 +102,78 @@ Let's take a look at the official Chrome V8 website: [https://developers.google.
+*Event Loop via [link](https://i.imgur.com/rnQEY7o.png)*
 
-I know it looks pretty weird, but let's break down the different parts of the picture and see if we can get it to make more sense. For now, disregard the heap part.
+Let's break down the different parts of the picture and see if we can get it to make more sense. For now, disregard the heap part.
 
-The outer box is the Google Chrome that we know and love. For this example that is our programming environment.
+- The outer box is the Google Chrome that we know and love. For this example that is our programming environment.
+![Event Loop Chrome](assets/eventloopchrome.png)
 
-The next box we see is the JS box. This is where our code runs. The call stack is where we are in the code. We can only push and pop things onto our stack. This represents the single thread provided.  
+- The next box we see is the JS box. This is where our code runs. The call stack is where we are in the code. We can only push and pop things onto our stack. This represents the 🧵 `single-thread` 🧵 provided.
+![Event Loop Javascript](assets/eventloopjavascript.png)
 
-Our WebAPIs, is where the _magic_ happens. This is where our DOM lives, and our asynchronous calls such as SetTimeout, SetInterval, AJAX calls, and our event listeners. The WebAPIs are effectively threads in our JS.  
+- Our WebAPIs, is where the 🔮*magic*🔮 happens. This is where our DOM lives, and our asynchronous calls such as SetTimeout, SetInterval, AJAX calls, and our event listeners. The WebAPIs are effectively threads in our JS. 
+![Event Loop Javascript](assets/eventloopwebapis.png)
 
 When we hit an asynchronous call in our stack, the call get's moved over to the WebAPI's area until it resolves.
 
-This means that if our code has a setTimeout with a time of 5 seconds. That call will move over to the WebAPI and wait for 5 seconds.
+This means that if our code has a `setTimeout` with a time of 5 seconds. That call will move over to the WebAPI and wait for 5 seconds.
+![eventLoopImage](assets/eventloop.png)
 
-Once a WebAPI has resolved, it then moves into the callback queue. A queue means first in, first out. Think about it like waiting in line. It's like only being able to use `shift` and `push` with an array.
+Once a WebAPI has resolved, it then moves into the callback queue. A queue means first in, first out. Think about it like waiting in line. Or like only being able to use `.shift` and `.push` with an array.
 
 The items in the queue get resolved only once the call stack is clear. Once the stack is clear it will take the first item from the queue and put it into the stack. Once the stack is clear again, the process will be repeated.
 
-This circular motion of stack, to WebAPI, to Callback queue, to stack is the _Event Loop_.
+This circular motion of stack, to WebAPI, to Callback queue, to stack is the 🌀 _Event Loop_.🌀
 
 Using this information let's see if we can predict the order that things will occur:
```

Comment:
I don't know if the previous visual is your own work, or if you have the ability to create a similar image. 

I would leave the previous image with the broad overview given. 

Because this class is currently being taught remotely, the instructors are at a disadvantage when it comes to trying to whiteboard/draw, so anything you can do to support with more visuals can be very helpful.

 I would make a separate image with exactly the code for the Yoda example.

Reuse the image again and again, leave the specific part being discussed in color, and grey out the other parts.

I would also not include the heap, since it is not being discussed.

Reply:
👍